script/arbitrum-sepolia/deployer.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.22;
+
+import { ConfigHub } from "../../src/ConfigHub.sol";
+import { EscrowSupplierNFT } from "../../src/EscrowSupplierNFT.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SetupHelper } from "../setup-helper.sol";
+import { DeploymentHelper } from "../deployment-helper.sol";
+
+library ArbitrumSepoliaDeployer {
+    uint constant chainId = 421_614; // id for arbitrum sepolia
+    address constant USDC = 0x69fC9D4d59843C6E55f00b5F66b263C963214C53; // CollarOwnedERC20 deployed on 12/11/2024
+    address constant WETH = 0xF17eb654885Afece15039a9Aa26F91063cC693E0; // CollarOwnedERC20 deployed on 12/11/2024
+    address constant WBTC = 0x19d87c960265C229D4b1429DF6F0C7d18F0611F3; // CollarOwnedERC20 deployed on 12/11/2024
+    address constant swapRouterAddress = address(0x101F443B4d1b059569D643917553c771E1b9663E);
+
+    address constant sequencerUptimeFeed = address(0);
+
+    uint24 constant oracleFeeTier = 3000;
```

Comment:
> what does the fee tier have to do with the twap window?

nothing, both are not what we're using on mainnet. For TWAP it's important, for fee tier it isn't as important, but does also cause differences (for example in "slippage" tolerances, since we count the swap fee as "slippage" in tests)

> also, used 3000 cause i deployed the assets using 3000 💀

I guess worth changing in the scripts for next time. Could probably change it in the deployer, and use those values in the fork tests that deploy (for sepolia). But leave 3000 as what's deployed already (and tested in fork tests that don't deploy - but only load).